apps/rush/src/utilities/RushHooksManager.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  RushHooks,
+  Utilities,
+  RushHookName
+} from '@microsoft/rush-lib';
+
+export default class RushHooksManager {
+  public constructor(private _rushHooks: RushHooks) {
+  }
+
+  public handle(hookName: RushHookName): void {
+    this._rushHooks.get(hookName).forEach((script) => {
+      Utilities.executeCommandOnShell(script,
```

Comment:
I'm concerned that if a poorly written script is printing errors, people will report problems to the Rush maintainers, because it looks like Rush is failing.

---
In reply to: [117124061](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r117124061) [](ancestors = 117124061,117121270)